### PR TITLE
Add SKA/FASTA module

### PR DESCRIPTION
**Pull request by itrujnara**
## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [x] Add a resource `label`
- [x] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - For modules:
    - [x] `nf-core modules test <MODULE> --profile docker`
    - [ ] `nf-core modules test <MODULE> --profile singularity`
    - [ ] `nf-core modules test <MODULE> --profile conda`

### CODE REVIEW

**Review by JoseEspinosa — APPROVED** [review]
LGTM! Only a couple of small suggestions, once the failing test is fixed feel free to merge! 🚀

File: nxf-tmp.xaQb2l [hunk 1]
```diff
@@ -0,0 +1,511 @@
+#!/usr/bin/env bash
```
[the remark this answers is not in the document]
Comment (itrujnara) [reply nxf-tmp.xaQb2l:1]: Probably, not sure how it got here

File: modules/nf-core/ska/fasta/main.nf [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+    script:
+    def args    = task.ext.args ?: ''
+    def prefix  = task.ext.prefix ?: "${meta.id}"
+    def arg_list = fasta_list ? "-f ${fasta_list}" : ''
+    """
+    ska fasta \\
+        ${args} \\
+        ${fastas} \\
+        ${fasta_list} \\
```
Comment (sateeshperi) [inline modules/nf-core/ska/fasta/main.nf:28]:
```suggestion
        ${arg_list} \\
```
am i right here?

**Review by sateeshperi — CHANGES_REQUESTED** [review]

File: modules/nf-core/ska/fasta/main.nf (hunk 2 above)
Comment (itrujnara) [reply modules/nf-core/ska/fasta/main.nf:28]: My bad, it's obviously a typo, `arg_list` should go here

**Review by sateeshperi — APPROVED** [review]

**Pull request merged**